File: LoanCalculatorAPI/Models/Loan.cs

namespace LoanCalculatorAPI.Models
{
    public class Loan
    {
        public int Period { get; set; }
        public double Amount { get; set; }
        public int LoanType { get; set; }

    }
    public enum LoanType
    {
        Standard = 10,
        Mortgage = 20,
        Carloan = 30,
        CreditCard = 40
    }
}

File: VismaLoanCalculator/src/app/_models/loan.ts
import { provideProtractorTestingSupport } from "@angular/platform-browser";
import { LoanType } from "../loantype/loantype.component";

export interface ILoan {
    period: number;
    amount: number;
    interest: number;
    type: LoanType;
}

export class Loan implements ILoan {
    period: number;
    amount: number;
    interest: number;
    type: LoanType;

    constructor() {

        this.period = 1;
        this.amount = 1000;
        this.interest = 3.5;
        this.type = LoanType.Standard;
    }

}